docs/source/getting_started/index.rst

```diff
@@ -44,29 +44,17 @@ Starting PostgreSQL Container
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Now we need to run ``PostgreSQL`` in a container, attach it to the network you
-have created before, and expose ports for communication:
+have created before:
 
 .. code-block:: shell
 
   docker run --name some-postgres \
   -e POSTGRES_USER=postgres \
   -e POSTGRES_PASSWORD=mysecretpassword \
-  -p 5432:5432 \
   --network=iroha-network \
   -d postgres:9.5
 
-.. note:: If you already have Postgres running on a host system on default port
-  (5432), then you should pick another free port that will be occupied. For
-  example, 5433: ``-p 5433:5432 \``
-
-Creating Blockstore
```

Comment:
Please explain why don't we need a blockstore anymore 🙏

Reply:
Docker creates volume automatically when running `docker run` if it does not exist.